### PR TITLE
VBLOCKS-3815 Update README.md

### PR DESCRIPTION
**Pull request by charliesantos**
**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ ] I acknowledge that all my contributions will be made under the project's license.

VBLOCKS-3815 Update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,62 @@
...
+
+## Run the project locally
+
+1. Start the local server under the `twilio-voice-js-reference-components` folder.
+
+```bash
+npm start
+```
+
+2. Access the following components under the following urls.
```
Comment (mhuynh5757) [inline README.md:59]:
```suggestion
2. Access the following components under the following URLs.
```

File: README.md [hunk 2]
```diff
@@ -1,21 +1,62 @@
...
+
+- Dialer related components, access [http://localhost:3030/twilio-voice-dialer](http://localhost:3030/twilio-voice-dialer).
```
Comment (mhuynh5757) [inline README.md:61]: Something about putting localhost into the readme feels kind of weird. What if we just have the express server print out something instead, like "Reference components ready! View each component: LINKS"

Comment (charliesantos) [reply README.md:61]: Not sure what's weird about it. I see it most of the time. Do you have anything specific why you think this is weird?

Comment (mhuynh5757) [reply README.md:61]: I personally have not seen it much. I don't think there's anything wrong with it persay, but I do think that usually I don't see it in READMEs and I see the terminal print it out when ready. Vite, Metro (React Native), webpack, do the terminal thing from what I've seen.

Comment (charliesantos) [reply README.md:61]: It sounds like it's a developer preference. I'll remove/update it if you find any downsides to this.

[the remark this answers is not in the document]
Comment (charliesantos) [reply README.md:61]: good catch

**Review by kpchoy — APPROVED** [review]

**Comment by charliesantos** [comment]
Good catch on the double white spaces @mhuynh5757 !

**Pull request merged**